doc/z-chap10.xml

```diff
@@ -179,4 +180,15 @@ x1x2x2^-1x1^-1x1x2]]></Example>
 
     <#Include Label = "GreensDClassOfElement" >
   </Section>
+
+
+  <Section>
+    <Heading> Words and strings in finitely presented semigroups
+    </Heading>
+
```

Comment:
Do you think some explanation of what GAP "words" are and how they are used could be added here, for context? E.g. specify that by a "word", you mean a list of pos ints, and that there are methods for converting arbitrary words to "nice" GAP lists.